Explorer/Assets/Scripts/ECS/StreamableLoading/Cache/IStreamableCache.cs

```diff
@@ -26,15 +26,42 @@ public interface IStreamableCache<TAsset, TLoadingIntention> : IEqualityComparer
 
         void Add(in TLoadingIntention key, TAsset asset);
 
-        /// <summary>
-        ///     Signal the cache that a single usage of asset went out of scope.
-        ///     It is needed for cache with limited capacity based on LRU, reference counting
-        /// </summary>
-        void Dereference(in TLoadingIntention key, TAsset asset);
-
         /// <summary>
         ///     Unload assets from the cache to free memory
         /// </summary>
         void Unload(IPerformanceBudget frameTimeBudget, int maxUnloadAmount);
+
+        class Fake : IStreamableCache<TAsset, TLoadingIntention>
+        {
+            public bool Equals(TLoadingIntention x, TLoadingIntention y) =>
+                throw new Exception("I am fake");
+
+            public int GetHashCode(TLoadingIntention obj) =>
+                throw new Exception("I am fake");
```

Comment:
More informative messages are optional and could be added. I don't think it should be renamed to FakeCache because to create this object it is required call outer Cache class first: new IStreamableCache<T,TK>.Fake()